Dockerfile.test

```diff
@@ -1,17 +1,16 @@
-FROM golang:1.17-alpine
+FROM golang:1.19-alpine
 
 MAINTAINER Conjur Inc
 
-
 RUN apk add --no-cache bash \
                        build-base \
                        curl \
                        git \
                        jq \
                        less && \
-    go get -u github.com/jstemmer/go-junit-report && \
-    go get -u github.com/axw/gocov/gocov && \
-    go get -u github.com/AlekSi/gocov-xml && \
+    go install github.com/jstemmer/go-junit-report@latest && \
+    go install github.com/axw/gocov/gocov@latest && \
+    go install github.com/AlekSi/gocov-xml@latest && \
```

Comment:
I am wary of using `@latest` and potentially unintended consequences. How about we pin to specific versions instead ?

Reply:
Given that this is only for the test image, I think it might be worth the tradeoff to use latest (which is what we were doing before this change with `go get -u`)